api/collection-view.yaml

```diff
@@ -7,320 +7,328 @@ description:
-      
+
   getEmptyView:
     description: |
       ...
-      
+
       @api public
-    
+
     examples:
-      
+
   addEmptyView:
     description: |
       ...
-      
+
       @api public
       @param {Marionette.Model} child
       @param {Marionette.View} EmptyView
-    
+
     examples:
-      
+
   getChildView:
     description: |
       ...
-      
+
       @api public
       @param {Marionette.Model} child
-    
+
     examples:
-      
+
   addChild:
     description: |
       ...
-      
+
       @api public
       @param {Marionette.Model} child
       @param {Marionette.View} ChildView
       @param {Number} index
-    
+
     examples:
-      
+
   _updateIndices:
     description: |
       ...
-      
+
       @api private
       @param {Marionette.View} view
       @param {Boolean} increment
       @param {Number} index
-    
+
     examples:
-      
+
   _addChildView:
     description: |
       ...
-      
+
       @api private
       @param {Marionette.View} view
       @param {Number} index
-    
+
     examples:
-      
+
   renderChildView:
     description: |
       ...
-      
+
       @api public
       @param {Marionette.View} view
       @param {Number} index
-    
+
     examples:
-      
+
   buildChildView:
     description: |
       ...
-      
+
       @api public
       @param {Marionette.Model} child
       @param {Marionette.View} ChildViewClass
       @param {Object} childViewOptions
-    
+
     examples:
-      
+
   removeChildView:
     description: |
       ...
-      
+
       @api public
       @param {Marionette.View} view
-    
+
     examples:
-      
+
   isEmpty:
     description: |
       ...
-      
+
       @api public
       @param {Marionette.Collection} collection
-    
+
     examples:
-      
+
   checkEmpty:
     description: |
       ...
-      
+
       @api public
-    
+
     examples:
-      
+
   attachBuffer:
     description: |
       ...
-      
+
       @api public
       @param {Marionette.Collectionview} collectionView
       @param {} buffer
-    
+
     examples:
-      
+
   attachHtml:
     description: |
       ...
-      
+
       @api public
       @param {Marionette.Collectionview} collectionView
       @param {Marionette.View} childView
       @param {Number} index
-    
+
     examples:
-      
+
   _insertBefore:
     description: |
       ...
-      
+
       @api private
       @param {Marionette.View} childView
       @param {Number} index
-    
+
     examples:
-      
+
   _insertAfter:
     description: |
       ...
-      
+
       @api private
       @param {Marionette.View} childView
-    
+
     examples:
-      
+
   _initChildViewStorage:
     description: |
       ...
-      
+
       @api private
-    
+
     examples:
-      
+
   destroy:
     description: |
       ...
-      
+
       @api public
-    
+
     examples:
-      
+
   destroyChildren:
     description: |
       ...
-      
+
       @api public
-    
+
     examples:
-      
+
```

Comment:
For the life of me, I am trying to avoid these changes. Maybe next try will get it.